src/umpire/CMakeLists.txt

```diff
@@ -170,6 +170,7 @@ install(TARGETS
 
 install(FILES
   ${umpire_headers}
+  ${umpire_device_headers}
```

Comment:
There should be a separate `install` command with the rest of the `umpire_device` library stuff (inside the `if (ENABLE_CUDA)` block)

Reply:
Oh duh, yea you're right